_tests/tests/__snapshots__/ApiTest__test_api_valid_features__1.php

```diff
@@ -0,0 +1,1416 @@
+                                {
+                                    "title": "can batch create webhooks",
+                                    "status": "passed"
+                                },
+                                {
+                                    "title": "can batch update webhooks",
+                                    "status": "passed"
+                                },
+                                {
+                                    "title": "can batch delete webhooks",
+                                    "status": "passed"
+                                }
+                            ]
+                        }
+                    }
+                ],
+                "summary": "Test Suites: 0 skipped, 0 failed, 20 passed, 20 total | Tests: 2 skipped, 0 failed, 256 passed, 258 total."
+            }
+        },
+        {
+            "debug_log": [
+                {
+                    "count": "4287",
+                    "message": "PHP Warning: array_filter() expects parameter 1 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Internal\\/Admin\\/FeaturePlugin.php on line 211"
+                },
+                {
+                    "count": "4287",
+                    "message": "PHP Warning: array_keys() expects parameter 1 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Internal\\/Admin\\/FeaturePlugin.php on line 211"
+                },
+                {
+                    "count": "3676",
+                    "message": "PHP Warning: array_diff(): Expected parameter 1 to be an array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 184"
+                },
+                {
+                    "count": "3676",
+                    "message": "PHP Warning: array_values() expects parameter 1 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 184"
+                },
+                {
+                    "count": "3676",
+                    "message": "PHP Warning: in_array() expects parameter 2 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 195"
+                },
+                {
+                    "count": "611",
+                    "message": "PHP Warning: Invalid argument supplied for foreach() in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 135"
+                },
+                {
+                    "count": "4251",
+                    "message": "PHP Notice: New Product Editor is enabled as expected. in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce-product-feeds\\/woocommerce-product-feeds.php on line 13"
```

Comment:
Awesome, we can see that it's activated here. However, there are some other notices being thrown around, I wonder if it has something to do with the way we enabled the feature plugin in the SUT?

```json
                   "count": "4287",
                    "message": "PHP Warning: array_filter() expects parameter 1 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Internal\\/Admin\\/FeaturePlugin.php on line 211"
                },
                {
                    "count": "4287",
                    "message": "PHP Warning: array_keys() expects parameter 1 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Internal\\/Admin\\/FeaturePlugin.php on line 211"
                },
                {
                    "count": "3676",
                    "message": "PHP Warning: array_diff(): Expected parameter 1 to be an array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 184"
                },
                {
                    "count": "3676",
                    "message": "PHP Warning: array_values() expects parameter 1 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 184"
                },
                {
                    "count": "3676",
                    "message": "PHP Warning: in_array() expects parameter 2 to be array, null given in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 195"
                },
                {
                    "count": "611",
                    "message": "PHP Warning: Invalid argument supplied for foreach() in \\/var\\/www\\/html\\/wp-content\\/plugins\\/woocommerce\\/src\\/Admin\\/Features\\/Features.php on line 135"
                },
```

Reply:
Turns out it was because I wasn't using `add_filter` and I wasn't returning the `$features` array.